knowledge-base/kb-changehandicon-to-loadingicon.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Change Hand Icon to Any Loading Icon Upon Clicking on Filter
+description: "How to change hand icon to any loading icon upon clicking on filter"
+type: how-to
+page_title: How to change hand icon to any loading icon upon clicking on filter for WPF 
+slug: kb-changehandicon-to-loadingicon
```

Comment:
The slug should contain the name of the control in lowercase and each word should be separated with a hyphen "-".